src/layout.css

```diff
@@ -224,6 +225,10 @@
 .absolute { position: absolute !important; }
 .relative { position: relative !important; }
 .static { position: static !important; }
+.sticky {
+  position: sticky !important;
+  position: -webkit-sticky !important; /* stylelint-disable-line value-no-vendor-prefix */
```

Comment:
Safari still requires this. Unless there's a different way to handle adding this, I think disabling the style lint rule here makes sense.

Reply:
We use the autoprefixed postCSS plugin so there's no need for prefixes: https://github.com/mapbox/assembly/blob/publisher-staging/scripts/build-css.js#L114-L116